tests/fs/test_memfs_fsync.c

```diff
@@ -25,22 +23,20 @@ int main() {
   struct stat st;
 
   // a file whose contents are just 'az'
-  if ((stat("/wakaka.txt", &st) != -1) || (errno != ENOENT))
-    result = -1000 - errno;
+  rtn = stat("/wakaka.txt", &st);
+  assert(rtn != -1 || errno == ENOENT);
```

Comment:
This doesn't look right. It should flipped I think,
```suggestion
  assert(rtn == -1 && errno == ENOENT);
```